frontend/src/translations/en.json

```diff
@@ -102,17 +102,17 @@
   "account.login.connect": "Conect with your existing account",
   "task.assignment.remove.action": "Remove",
   "task.assignment.remove.user": "Unassign user",
-  "task.assignment.remove.reason": "Why this user will not work on this task anymore?",
+  "task.assignment.remove.reason": "Why this user won't work on this task anymore?",
   "task.assignment.actions.cancel": "Cancel",
   "task.assignment.actions.apply": "Apply",
   "account.dialog.status": "Task status",
-  "task.status.deadline.subheading": "Choose a date that this task should be finished",
+  "task.status.deadline.subheading": "Choose a deadline for this task",
   "task.status.deadline.day.label": "Day",
   "task.status.deadline.day.insert.label": "Choose a date",
-  "task.status.deadline.set.target": "set to target date to {date}",
+  "task.status.deadline.set.target": "Set a date to target {date}",
```

Comment:
Hi Alexandre! Sorry, I had not understood where it would show and the message it should send. If it's not right, you can talk to me again and I'm going to correct it.

I think it should be:
Set the target date to {date}